timemachine/fe/mle.py

```diff
@@ -212,3 +214,103 @@ def infer_node_vals_and_errs(
     dg_err = bootstrap_estimates.std(0)
 
     return dg, dg_err
+
+
+def infer_node_vals_and_errs_networkx(
+    nx_graph: nx.Graph,
```

Comment:
Does that work expectedly with a DiGraph vs nx.Graph? I believe the subsetting might behave differently with those two cases.

Reply:
You're right, it should be `DiGraph`. Changed in https://github.com/proteneer/timemachine/pull/987/commits/0e30af40ed9ce764a75df5ac27289d54ab09c6d3